cmd/jaeger/internal/integration/badgercleaner/.nocover

```diff
@@ -0,0 +1 @@
+FIXME
```

Comment:
are you planning to add tests in this PR?

Reply:
yes, i am working on it.